phpstan.neon

```diff
@@ -9,11 +9,9 @@ parameters:
     ignoreErrors:
         - '#Expression on left side of \?\? is not nullable\.#'
```

Comment:
This also looks too broad.

Reply:
Also not needed, as the code can be simplified here

Reply:
Fixed.